app/Http/Controllers/AssignTaskUserController.php

```diff
@@ -34,4 +34,25 @@ public function assign(Request $request)
 
         return response()->json($response);
     }
+
+    public function remove(Request $request)
+    {
+        $task = $this->taskService->find($request->task_id);
+
+        $data = array();
+        $data['status_id'] = $task['status_id'];
```

Comment:
Line 43 to 50 are not needed, since you are only removing the assigned user.
You can remove them because It will update things that are not needed
and line 50 sets the array to empty so there will be nothing for line 52 to remove from